src/task2/resultFile.txt

```diff
@@ -0,0 +1,5 @@
+{"Book":"The Compound Effect","Author":"Darren Hardy","Amount":"5","Price":"9.48"}
```

Comment:
Amount field is redundant

Reply:
Also, pay attention to the keys

Reply:
CSV parser params were added for ignoring the Amount field and renaming headers